src/service/xhr-impl.js

```diff
@@ -47,6 +47,13 @@ const allowedMethods_ = ['GET', 'POST'];
 /** @private @const {!Array<function(*):boolean>} */
 const allowedJsonBodyTypes_ = [isArray, isObject];
 
+/** @private @enum {number} Allowed fetch responses. */
+const allowedFetchTypes_ = {
+  document: 1,
```

Comment:
We generally use `NAME: "VALUE"`, where `NAME` is some name you would identify by and `VALUE` is the normalized value to be used.